client/src/components/formlabels/FormLabels.tsx

```diff
@@ -61,11 +64,12 @@ const InputLabel = ({ text, isRequired = false }: InputLabelProps) => {
   );
 };
 
-const LoadingLabel = ({ text }: LabelProps) => {
+const LoadingLabel = ({ className, text }: LabelProps) => {
   return (
-    <FormText className="loading-label">
-      <span>{text}</span>
-      <Loader inline size={16} />
+    <FormText className={cx(className, "loading-label")}>
+      <span>
+        {text} <Loader inline size={16} />
```

Comment:
We should avoid inserting extra `[space]` characters:

```suggestion
        {text}
        <Loader className="ms-1" inline size={16} />
```